lib/upgrade-db.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+var MeshServer = require('strong-mesh-models').meshServer;
+var MeshServiceManager = require('strong-mesh-models').ServiceManager;
+var SQLite3 = require('loopback-connector-sqlite3');
+var async = require('async');
+var debug = require('debug')('upgrade-db');
+var fs = require('fs');
+var path = require('path');
+
+module.exports = checkUpgradeDb;
+
+function checkUpgradeDb(baseDir, memoryDbPath, dryRun, callback) {
```

Comment:
This name isn't used outside of this file, so it shouldn't matter too much.. but I find this name confusing in that it isn't checking something and returning the result of that check. Would it be better named `ensureUpgraded` or `maybeUpgradeDb`?